packages/playwright-ct-angular/index.d.ts

```diff
@@ -25,7 +25,7 @@ import type {
 } from '@playwright/test';
 import type { JsonObject } from '@playwright/experimental-ct-core/types/component';
 import type { InlineConfig } from 'vite';
-import type { Type } from '@angular/core';
+import type { Provider, Type } from '@angular/core';
 
 export type PlaywrightTestConfig<T = {}, W = {}> = Omit<BasePlaywrightTestConfig<T, W>, 'use'> & {
```

Comment:
`export type PlaywrightTestConfig` could be removed and could be exported from `@playwright/experimental-ct-core` since: https://github.com/microsoft/playwright/pull/28816/files#diff-748550f0be717ccc179e648fe9bf85fe98958da68a3559b9d46a7515b61774c2L29-L36. The dependency on Vite inside `@playwright/experimental-ct-angular` is no longer necessary because of this.

Reply:
Nice catch! Thanks.
Fixed